packages/app/components/Nav/NavBar.tsx

```diff
@@ -123,12 +124,24 @@ export default function NavBar({ courseId }: NavBarProps): ReactElement {
     null
   );
   const [visible, setVisible] = useState<boolean>(false);
+  const [updateModalVisible, setUpdateModalVisible] = useState<boolean>(true);
   const { pathname } = useRouter();
   const { course } = useCourse(courseId);
   const role = useRoleInCourse(courseId);
 
   const openQueues = course?.queues?.filter((queue) => queue.isOpen);
 
+  const addMember = async () => {
+    setUpdateModalVisible(false);
+    await API.organizations.addMember(profile.id, 1);
```

Comment:
I am not sure how you plan to make this work in the final implementation but the '1' here is not constant (even in development environment).

Reply:
The goal is to migrate existing users to organizational account, which is currently only one - UBCO. Thus, the PK of this organization is hard-coded.